byzcoin/bcadmin/main.go

```diff
@@ -107,6 +108,12 @@ var cmds = cli.Commands{
 		Usage:   "interact with byzcoin for debugging",
 		Aliases: []string{"d"},
 		Subcommands: cli.Commands{
+			{
+				Name:      "follow",
```

Comment:
Can we find another term for this ? I got confused with the "Follow" feature of the skipchain service. I'm not really good at naming so I let you think about it but it feels more like an integrity check.

Reply:
s/follow/replay/